### ISSUE TITLE
Investigate preloading images serially

### ISSUE THREAD

**Issue by chrismytton** [everypolitician/gender-balance#212]
Before #211 we were preloading all of the images for a term, but this happened in parallel and was very slow. We now only load the first two images, then when a card is swiped we add another card to the back of the deck and preload the image for that. This is ok but it means that if you're going through the game quite fast then it can be a bit laggy.

One possible improvement would be to preload the images serially, i.e. attach an "on load" event listener to the images, and when one finishes preloading start preloading the next one back and so on. Then the user will always see the first two images immediately, and any other images can be preloaded as the connection speed allows.

**Comment by andylolz** [comment]
But if the user bails out before completing the set, they’ve loaded lots of images unnecessarily.

Perhaps serial preload 4 images ahead (instead of the current 1 image ahead)?
